winres/winres.json

```diff
@@ -0,0 +1,54 @@
+{
+  "RT_GROUP_ICON": {
+    "OTHER": {
+      "0000": "hazelcast-icon-flat-sm.png"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "identity": {
+          "name": "",
+          "version": ""
```

Comment:
Do we need to set these?

Reply:
These are the default values created when you run "go-winres init". I left them unset so that we can just see the fields if we want to change anything